core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -1324,13 +1324,15 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
     self.fork >>= (_.toFutureWith(f))
 
   /**
-   * Converts this ZIO to [[zio.Managed]].
+   * Converts this ZIO to [[zio.Managed]]. This ZIO and the provided release action
```

Comment:
This is very nice. Makes it far easier to understand interruption semantics without digging through ZManaged / bracket